website/docs/integrations/vscode.mdx

```diff
@@ -6,11 +6,12 @@ description: ConfigCat Visual Studio Code extension. This is a step-by-step guid
 
 <a href="https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags" target="_blank">ConfigCat's Visual Studio Code extension</a> to manage feature flags from Visual Studio Code.
 
-## Feature overview
+Connect your Product and Config to your Workspace. Find your Feature Flag's usages in your code easily.
+Turn features On/Off right from VSCode. You can also easily modify the linked flags to edit or add new Targeting or Percentage Rules.
 
-<img src="/docs/assets/vscode/usage.gif" className="zoomable" alt="Usage of ConfigCat Feature Flags Visual Studio Code Extension" />
+This guide will help you with the Extensions installation and familiarise you with the Extensions usage.
```

Comment:
Javaslom, hogy "VSCode Extension" (vagy "Visual Studio Code Extension") szerepeljen "Extension" helyett. Hogy a nem-nagy-tudású userek is tudják, hogy mikor van szó a ConfigCatről és mikor van szó a másik toolról.